### PR TITLE
Update Algorithm Everything Be True

### PR DESCRIPTION
**Pull request by koustuvsinha**
Refer #1094

### CODE REVIEW

File: Algorithm-Everything-Be-True.md [hunk 1]
```diff
@@ -24,7 +37,7 @@ You could use loops or callbacks functions, there are multiple ways to solve thi
 
 **Solutions ahead!**
 
-## Code Solution using for-in loop & hasOwnProperty:
+## :beginner: Beginner : Code Solution using for-in loop & hasOwnProperty:
```
Comment (atjonathan) [inline Algorithm-Everything-Be-True.md]: ~~## :beginner: Beginner : Code Solution using for-in loop & hasOwnProperty:~~ ## :beginner: Beginner Code Solution (using for-in loop & hasOwnProperty):

File: Algorithm-Everything-Be-True.md [hunk 2]
```diff
@@ -99,8 +122,20 @@ truthCheck([{"user": "Tinky-Winky", "sex": "male"}, {"user": "Dipsy", "sex": "ma
...
 
-## Credits:
+#### Relevant Links
+
+- [JS Reduce Made Easy](JS-Reduce-Made-Easy)
+- [JS Array Reduce](JS-Array-Prototype-Reduce)
+
+## :trophy: Credits:
 
 If you found this page useful, you can give thanks by copying and pasting this on the main chat: **`thanks @Rafase282 @estevanmaito @HermanFassett for your help with Algorithm: Everything Be True`**
```
Comment (atjonathan) [inline Algorithm-Everything-Be-True.md]: Add two lines between `on the main chat:` and `Thanks`.
```diff
- If you found this page useful, you can give thanks by copying and pasting this on the main chat: **`thanks @Rafase282 @estevanmaito @HermanFassett for your help with Algorithm: Everything Be True`**
+ If you found this page useful, you can give thanks by copying and pasting this on the main chat:
+
+ **`thanks @Rafase282 @estevanmaito @HermanFassett for your help with Algorithm: Everything Be True`**
```

File: Algorithm-Everything-Be-True.md [hunk 3]
```diff
@@ -99,8 +122,20 @@ truthCheck([{"user": "Tinky-Winky", "sex": "male"}, {"user": "Dipsy", "sex": "ma
...
 
-> **NOTE:** Please add your username only if you have added any **relevant main contents** to the wiki page. (Please don't remove any existing usernames.)
+:clipboard: NOTES FOR CONTRIBUTIONS:
```
Comment (Rafase282) [inline Algorithm-Everything-Be-True.md]:
```diff
-:clipboard: NOTES FOR CONTRIBUTIONS:
+## :clipboard: NOTES FOR CONTRIBUTIONS:
```

**Comment by Rafase282** [comment]
# LGTM :+1:

**Pull request merged**